go/src/koding/klient/machine/index/index.go

```diff
@@ -64,6 +64,11 @@ func NewIndexFiles(root string) (*Index, error) {
 			return nil
 		}
 
+		// Don't include OSX trashes to scanned files.
+		if info.Name() == ".Trash" && info.Mode().IsDir() && runtime.GOOS == "darwin" {
```

Comment:
Would it make sense to turn it into `isIgnored(fi os.FileInfo) bool` and extend for more ignored files, like vim's swap and temporary ones etc.